.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "bracketSpacing": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "semi": true,
+  "singleQuote": true
```

Comment:
Single quote is off by default, otherwise all of these re-declare default values. I'd just use the defaults from prettier directly and leave an empty object within this file, e.g. use double quotes and whatever else prettier recommends by default.

Reply:
I'll do that 👍